Factory/SharedCatalogFactory.php

```diff
@@ -65,14 +65,13 @@ public function getSharedCategoryCollection()
             $indexResource = $this->getSharedCatalogCategoryResource();
             $connection = $indexResource->getConnection();
 
-            $query = "
-                SELECT category_id, GROUP_CONCAT(CONCAT(customer_group_id, '_', permission) SEPARATOR ',') AS permissions
-                FROM {$indexResource->getMainTable()} 
-                WHERE customer_group_id IN (SELECT customer_group_id FROM shared_catalog) 
-                GROUP BY category_id;
-            ";
-
-            $this->sharedCategoryCollection = $connection->fetchPairs($query);
+            $select = $connection->select()
+                ->from($indexResource->getMainTable(), [])
+                ->columns('category_id')
+                ->columns(['permissions' => new \Zend_Db_Expr("GROUP_CONCAT(CONCAT(customer_group_id, '_', permission) SEPARATOR ',')")])
+                ->group('category_id');
```

Comment:
Is it expected that the WHERE condition hasn't been ported ?

Reply:
Updated!